pkg/opensearch/lucene_handler.go

```diff
@@ -138,6 +138,9 @@ func processTimeSeriesQuery(q *Query, b *es.SearchRequestBuilder, fromMs int64,
 
 	// iterate backwards to create aggregations bottom-down
 	for _, bucketAgg := range q.BucketAggs {
+		bucketAgg.Settings = utils.NewJsonFromAny(
+			bucketAgg.generateSettingsForDSL(),
```

Comment:
this was already being done for Lucene logs in `parseLogsQuery`, copying how this is in Elasticsearch